tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3055,7 +3055,7 @@ public function getShares(string $user, string $path):SimpleXMLElement {
 			$user,
 			$this->getPasswordForUser($user),
 			"GET",
-			$this->getSharesEndpointPath("?path=$path"),
+			$this->getSharesEndpointPath($path),
```

Comment:
this should be fine. any reasons to change?
```suggestion
			$this->getSharesEndpointPath("?path=$path"),
```